libs/core/src/scss/interaction-state/_hover.scss

```
@@ -1,9 +1,11 @@
 @use 'layer';
+@use 'utilities';
 @use '../utils';
```

This double utils thing is a little unfortunate, but I see no other option until we have broken the original utils into appropriate modules. Thoughts, anyone?

---

I am talking about line 2-3, btw.

---

From the top of my head - without any deeper thought - would choosing a different namespace for the "global utils" module be an improvement?

```suggestion
@use 'utilities';
@use '../utils' as global-utils;
```

Usage example:

```
$stroke-width: utils.size('xxxxs');
```

becomes

```
$stroke-width: global-utils.size('xxxxs');
```

Not sure 🙂